test/parallel/test-repl-.paste.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+const terminalCode = '\u001b[1G\u001b[0J> \u001b[3G';
+
+function run(input, output, event) {
+  const stream = new common.ArrayStream();
+  let found = '';
+
+  stream.write = (msg) => found += msg.replace('\r', '');
+
+  const expected = `${terminalCode}.paste
```

Comment:
This code looks strange because of the indentation. Maybe change it to string concatenations.